bucket/windscribevpn-np.json

```diff
@@ -0,0 +1,41 @@
+{
+    "##": "This package is non-portable because the app requires installing drivers/services to work properly.",
+    "version": "2.4.11",
+    "description": "Open Source VPN with AdBlocking Proxy.",
+    "homepage": "https://windscribe.com/",
+    "license": "GPL-2.0",
+    "notes": [
+        "If windscribevpn-np is not installed properly, please run:",
+        "scoop uninstall windscribevpn-np"
+    ],

```

Comment:
```suggestion
```

I don't think this is necessary, because this is generally the rule for all *Scoop* apps.